pkgs/development/python-modules/apsw/default.nix

```diff
@@ -38,7 +39,7 @@ buildPythonPackage rec {
     "testVFSWithWAL"
     # no lines in errout.txt
     "testWriteUnraisable"
-  ];
+  ] ++ lib.optionals stdenv.isDarwin [ "testzzForkChecker" ];
```

Comment:
It is weird that this test starts to fail now. Two days ago in #334484 `apsw` didn't have any issues for `aarch64-darwin`..

Reply:
No, it is broken. I think I will create a separate PR.

https://hydra.nixos.org/build/269319976